app/livechat/server/methods/sendMessageLivechat.js

```diff
@@ -24,10 +24,22 @@ Meteor.methods({
 			},
 		});
 
+		const fields = {
+			t: 1,
+			closedAt: 1,
+			open: 1,
+		};
+
 		if (!guest) {
 			throw new Meteor.Error('invalid-token');
 		}
 
+		// Check if conversation is closed or not.
+		const closed = Rooms.findOneLivechatById(rid, fields);
```

Comment:
We can't add this validation here because the current version of the Livechat client needs to create a new room when the current room is closed.
The point is that the issue #13733 is related to REST API endpoints, and our new Livechat client works over this approach, so I suggest you add validation on the Livechat message endpoint:

https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/api/v1/message.js#L33

Reply:
ok, will do that.